src/EdFi.AnalyticsMiddleTier.Tests/Classes/StudentProgramCohortDim.cs

```diff
@@ -14,6 +14,8 @@ public class StudentProgramCohortDim
         public string StudentProgramCohortKey { get; set; }
         public string StudentSchoolProgramKey { get; set; }
         public string StudentSchoolKey { get; set; }
+        public string EntryGradeLevelDescriptor { get; set; }
```

Comment:
You correctly updated the should_match_column_dictionary test for these two new columns, but we should also include tests to verify expected values come back for these new columns. Take a look at `When_querying_the_StudentProgramCohortDim_view` and inside there should will see tests like `Then_should_have_StudentProgramCohortKey`, `Then_should_have_CohortDescription`, and `Then_should_have_ProgramName` amongst others. We should have similar tests for these two new fields, so you will need to make a new test case in that file for it, along with creating the needed xml files inside `TestCases\StudentProgramCohortDim` .